azure-pipelines.yml

```diff
@@ -170,6 +170,64 @@ stages:
               summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/coverage.xml'
               reportDirectory: '$(System.DefaultWorkingDirectory)/**/htmlcov'
 
+      - job: no_sqlalchemy
+        condition: or(eq(variables.isScheduled, true), eq(variables.isMain, true), eq(variables.isManual, true))
+
+        services:
+          postgres: postgres
+
+        variables:
+          GE_pytest_opts: ''
```

Comment:
If opts are empty, do we need this section?

Reply:
Good call - was just mirroring the comprehensive section but based on our conversation in slack this is probably better done with just a script that imports GE instead of running the full pytest suite anyway.